capstone/src/main/resources/static/index.html

```diff
@@ -3,11 +3,21 @@
   <head>
     <meta charset="UTF-8">
     <title>Events</title>
-    <link rel="stylesheet" href="style1.css">
+    <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
 
   <body>
+    <nav class="nav-bar">
+      <ul>
+        <li class="home"><a href="index.html">College Activity Finder</a></li>
+        <li><a href="">Events</a></li>
+        <li><a href="">Organizations</a></li>
+        <li class="active"><a href="">My Account</a></li>
+        <li><a href="=">About Us</a></li>
+      </ul>
+    </nav>
+
```

Comment:
we shoud try to use ThymeLeaf later on to reduce code duplication.

Reply:
Create an issue, so you can prioritize (or deprioritize!) it.